### PR TITLE
Added documentation for enabling Cloud9

### PR DESCRIPTION
**Pull request by sean-e-dietrich**
Added Cloud9 IDE documentation
Closes https://github.com/docksal/service-cli/issues/50

### CODE REVIEW

File: docs/tools/cloud9.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Then restart the project
+
+```bash
+fin project restart
+```
+
+## Accessing IDE
+
+After enabling the IDE can be accessed at `ide.PROJECT_NAME.docksal`.
```
Comment (shelane) [inline docs/tools/cloud9.md]: "After enabling,"

File: docs/tools/cloud9.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+!!! note "Configuration confinements"
+	Since vhost-proxy does not support routing more than a single custom port, IDE mode cannot be used in conjunction
+	with the nodejs stack (stack-node.yml).
+
+
+!!! tip "Testing before committing"
+	To test the Cloud9 IDE before adding it to your project [click here](http://c9.github.io/core/).
```
Comment (shelane) [inline docs/tools/cloud9.md]: "click here" is evil. You can reword the sentence to something like:
"Test the Cloud9 IDE before adding it to your project." Link the "Test the Cloud9 IDE" in the sentence to the URL.

File: docs/tools/cloud9.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Cloud9 IDE
+
+Cloud9 is an open source, web based, IDE (integrated development environment) that allows for a developer to make changes
```
Comment (shelane) [inline docs/tools/cloud9.md]: The comma before IDE is not necessary as it is not a separate item in a list. Rather open source and web-based are descriptors of the IDE.

File: docs/tools/cloud9.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+### Set the `IDE_ENABLED` variable for the project.
+
+Enable IDE for the project by setting the `IDE_ENABLED` variable.
+
+```bash
+fin config set --env=local IDE_ENABLED 1
+```
+
+!!! tip "For Other Developers on the Project"
+	To enable for other developers within the project remove the `--env=local` option from the above command.
```
Comment (shelane) [inline docs/tools/cloud9.md]: ...project, remove...

File: docs/tools/cloud9.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+![Cloud9 IDE](../_img/cloud9-ide-screenshot.png)
+
+## Enabling Cloud9
+
+### Set the `IDE_ENABLED` variable for the project.
+
+Enable IDE for the project by setting the `IDE_ENABLED` variable.
+
+```bash
+fin config set --env=local IDE_ENABLED 1
```
Comment (achekulaev) [inline docs/tools/cloud9.md]: `IDE_ENABLED 1` => `IDE_ENABLED=1`

File: docs/tools/cloud9.md (hunk 4 above)
Comment (achekulaev) [inline docs/tools/cloud9.md]: I think it's shorter and cleaner to write:

To enable for other developers within the project:

```bash
fin config set IDE_ENABLED=1
```

**Comment by lmakarov** [comment]
Cloud9 is an experimental feature.

It requires `docksal/cli:edge-php7.1-ide` image, which is not yet ready to be released, as there have been changes done to get Cloud9 working, which introduce breaking changes for existing users and will need to be revised (see https://github.com/docksal/service-cli/issues/51).

We'll have to put this PR on hold until `docksal/cli` v2.3.0 is ready to be released.

**Pull request merged**